docs/reference/builder.md

```diff
@@ -817,14 +817,14 @@ change them using `docker run --env <key>=<value>`.
 
 ADD has two forms:
 
-- `ADD <src>... <dest>`
-- `ADD ["<src>",... "<dest>"]` (this form is required for paths containing
+- `ADD [--chown <user>:<group>] <src>... <dest>`
+- `ADD [--chown <user>:<group>] ["<src>",... "<dest>"]` (this form is required for paths containing
 whitespace)
 
 The `ADD` instruction copies new files, directories or remote file URLs from `<src>`
 and adds them to the filesystem of the image at the path `<dest>`.
 
-Multiple `<src>` resource may be specified but if they are files or
+Multiple `<src>` resources may be specified but if they are files or
 directories then they must be relative to the source directory that is
```

Comment:
directories, they must be

Also, "relative" I think means that they must be subdirectories of the build context? Maybe you can reword this to make it clearer.